deploy/cloudformation/create-stacks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Can you decorate it with `-e` like with other shell files?

Reply:
Sure.